tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj

```diff
@@ -40,6 +40,16 @@
     <PackageReference Include ="MySql.Data" Version="6.10.9" Condition="'$(TargetFramework)' == 'netcoreapp3.1'" />
     <PackageReference Include ="MySql.Data" Version="8.0.15" Condition="'$(TargetFramework)' == 'net5.0'" />
     <PackageReference Include ="MySql.Data" Version="8.0.30" Condition="'$(TargetFramework)' == 'net6.0'" />
+
+    <!-- MySqlConnector framework references -->
+    <PackageReference Include ="MySqlConnector" Version="1.1.0" Condition="'$(TargetFramework)' == 'net462'" />
```

Comment:
The old tests used to ONLY test version `1.0.1`, I think we should maintain coverage for that version.